tools/file_packager.py

```diff
@@ -716,6 +716,10 @@ def was_seen(name):
         }
       };
       xhr.onerror = function(event) {
+        if (window.location.protocol === "file:") {
+          if (Module['setStatus']) Module['setStatus']('CORS May not be allowed on a file: URL. Use `python -m SimpleHTTPServer` to create a server at this location.');
```

Comment:
I don't think we need both of these lines. Just the second one is enough - the user will need to look in the devtools for the details of the error (otherwise the page will just say "exception thrown"), but that seems reasonable.